src/Apple/Provider.php

```diff
@@ -101,12 +116,53 @@ public function getAccessTokenResponse($code)
      */
     protected function getUserByToken($token)
     {
-        static::verify($token);
+        $this->checkToken($token);
         $claims = explode('.', $token)[1];
 
         return json_decode(base64_decode($claims), true);
     }
 
+    protected function getClientSecret()
+    {
+        if (!$this->jwtConfig) {
+            $this->getJwtConfig(); // Generate Client Secret from private key if not set.
+        }
+
+        return $this->clientSecret;
+    }
+
+    protected function getJwtConfig()
+    {
+        if (!$this->jwtConfig) {
+            $private_key_path = config('services.apple.private_key', '');
```

Comment:
can we access this via the standard method, ie add to 

```
public static function additionalConfigKeys()
    {
        return ['private_key', 'otherkey...'];
    }
```

```
$this->getConfig('private_key', '')
```

Reply:
Yes, we can. Changed.